yarn.lock

```diff
@@ -4488,6 +4494,7 @@ __metadata:
     webpack: ^5.73.0
     webpack-bundle-analyzer: ^4.5.0
     webpack-cli: ^4.10.0
+    zenroom: ^2.2.3
```

Comment:
zenroom should not be a dependency, why it is here?
I think only the NaCl library was missing

Reply:
Zenroom is actually imported [here](https://github.com/tracking-exposed/trex/blob/master/packages/shared/src/providers/zenroom.provider.ts) even though the implementation is not used for real.

Reply:
it is a dependency we can discontinue, we might not use it even to address the zero knowledge signature schema

Reply:
Ok, removed it!